ui/plugins/eu.esdihumboldt.hale.ui.common/src/eu/esdihumboldt/hale/ui/common/definition/viewer/OptionalPropertiesFilter.java

```diff
@@ -97,4 +97,61 @@ private boolean areChildrenOptional(EntityDefinition entityDef) {
 		return true;
 	}
 
+	private Collection<? extends EntityDefinition> getChildren(EntityDefinition entityDef) {
```

Comment:
Sorry to comment again - I think it would be really helpful to have this method available in `AlignmentUtil`, as I think this is not the only place that this is done / needed.